js/src/threepio.js

```diff
@@ -10,39 +11,34 @@ export default class Threepio {
     this.tf = tf;
   }
 
-  normalizeFunctionName(name) {
-    return name
+  _normalizeFunctionName(name, toLang) {
+    const result = name
       .match(NORMALIZATION_REGEX)
       .join('')
       .toLowerCase();
-  }
 
-  matchArgs(args, argMap) {
-    // Sort args by index
-    argMap.args.sort((a, b) =>
-      a.index === b.index ? 0 : a.index < b.index ? -1 : 1
-    );
-    for (const [i, arg] of args.entries()) {
-      // TODO: match args based on fromLibrary name
-      // TODO: Ensure order is preserved
-      // TODO: Ensure there are not any missing required arguments
-      console.log(i, arg);
-      continue;
+    if (result in this.mappedCommands[toLang]) {
+      return result;
+    }
+
+    if (result in WORD_TRANSLATIONS) {
```

Comment:
It's strange that when checking against mappedCommands, we look at target framework (`toLang`),
but WORD_TRANSLATIONS seem to be generic, and also having opposite translations like
```
  eq: 'equal',
  equal: 'eq',
```
It seems only one of these translations is true, depending on toLang? :)
Also, it seems strange that we need separate constant for this, why it's not in mappedCommands?

Reply:
I left it out of mappedCommands as this was meant as really word level translations. These translations will be used for argument mapping and others as sometimes certain libraries lexicon is structurally the same but have different words.

For instance, pytorch prefers `eq` for describing equality where-as tensorflow prefers `equal`.
Or for many tensor operations, pytorch prefers `input`, and `other` as the two tensors fed in. Tensorflow prefers `a` and `b`